WISE-Community/WISE-Client
# feat(Table): Allow authors to import table

## Changes

Added the ability for authors to import a table in the Table component authoring.

## Test

1. Open a project in the Authoring Tool
2. Go to a Table step
3. Go to the Advanced view for the Table component authoring
4. Click the "Import Table" button
5. Choose a csv file from your computer (that has 500 or less cells)
6. Make sure the Table component now contains the data from your csv file
7. Try importing a table with more than 500 cells, it should not allow you to do so

Closes #684
